test/TestCommon.gmk

```diff
@@ -345,6 +345,15 @@ endif
 ifeq ($(UNAME_S), Linux)
   NUM_CORES := $(shell cat /proc/cpuinfo  | grep -c processor)
 endif
+ifeq ($(UNAME_S), FreeBSD)
```

Comment:
It might be worth collapsing these into

```
ifneq ($(findstring BSD,$(UNAME_S),))
  NUM_CORES := $(shell /sbin/sysctl -n hw.ncpu)
endif
```

Reply:
Good idea. I went with `ifeq ($(findstring BSD,$(UNAME_S)), BSD)` which I think is a bit easier to read and same as an existing findstring for BSD in the file.